docs/prfc/drafts/pimp_ver1.1.xml

```diff
@@ -0,0 +1,514 @@
+
+     <t>When server receives packet, it will place packets orderly, then it 
+     checks the sequence number of packet to determine if the packet is what 
+     it suppose to receive. Once the packet is accept by the server, it sends 
+     back a ACK packet.</t>
+
+     <t>On the other side, after send out the packet, the client will keep the 
+     data until it receives a corresponding ACK packet whose ACK number should 
+     be sequence number + 1. It keeps waiting for 10 seconds. After timeout, it 
+     will resend the packets starting from time out packet. Within the 10 seconds, 
+     it will only be regarded as latency of network.</t>
+     
+     <t>Additionally, out of extreme scenario, the protocol is simplified by 
+     sending back a retransmission request packet for every missing sequence 
+     number after the window has been filled. The window is identified as 
+     "filled" after a specified timeout, and at this time, the retransmit 
+     requests will be sent.In this case only the specific packets will be 
+     needed for retransmission.</t>
+     
+     <t>We no longer need a data offset field since the options field has been
+      removed, so the PIMP header will be of fixed length. Because the port 
+      numbers will be fixed, these fields can also be excluded from the PIMP header.</t>
+     
+     <t>Finally, we guard against SYN flooding by having a time out. If
+     nothing is received from a client within the timeout, the connection
+     is closed. SYN flooding may briefly occur, but the TCB will be flushed
+     of any connections that are not fully open each time the timeout elapses.</t>
+    
+    <section title="Reliability">
+    <t>The prime directive to PIMP is reliability. In order for data
+      to be reliably transmitted, sequence numbers are used. The sender
+      must complete the sequence and retransmit missing packets before
+      they are given a new window with the next set of sequence numbers.
+      Packets may be lost on the network, and a mechanism for retransmission
+      must be supplied. The sequence numbers prevent duplicates, verify all 
+      sent data has been received, and provide a mechanism for identifying 
+      missing segments. Checksums are provided to recognize corruption of data.
+      Retransmission of corrupted or fragmented data can then be requested.</t>
+    
+    <t>Additional meta data includes the "References" and "Author's Address"
+    sections. These elements MUST conform to RFC specifications.</t>
+    </section>
+  
+   </section>
+   
+   <section title="Protocol">
+     
+     
+      <!-- ARTWORK EXAMPLE -->
+   		<figure anchor="xml_packet" align="center">
+            <preamble>The following artwork demonstrates the structure of the PIMP packet:</preamble>
+            <artwork align="left">
+<![CDATA[
+    0                   1                   2                   3
```

Comment:
@ZhengyuanWang please see this comment from Robin and fix please. Then I'll merge changes.

Reply:
What should we get rid of? Do you mean the header diagram?

Reply:
Yes we should not have the header diagram because there are no bits...this is packet based. So we should just have the description of the python packet.

Reply:
will do